src/vocgui/models.py

```diff
@@ -235,6 +236,11 @@ class DocumentImage(models.Model):
         Document, on_delete=models.CASCADE, related_name="document_image"
     )
 
+    def image_tag(self):
+            return mark_safe('<img src="/media/%s" width="330" height="240"/>' % (self.image))
+
+    image_tag.short_description = ''
```

Comment:
And I'm wondering why this line is down here. I think it would be clearer to move it up to the other class properties? And the purpose of the line is not clear to me.

Reply:
![Screenshot from 2021-06-21 13-42-03](https://user-images.githubusercontent.com/75339002/122756369-9dd76b00-d296-11eb-98dd-933cade84fa3.png)
Added this line to remove "Image tag"

Reply:
![Screenshot from 2021-06-21 13-51-13](https://user-images.githubusercontent.com/75339002/122757316-c57b0300-d297-11eb-8a1f-d2425d5079c6.png)

Reply:
When adding new images, the preview should probably be empty instead of a broken image tag:
![Screenshot 2021-06-21 at 14-27-23 Vokabeln ändern Lunes CMS](https://user-images.githubusercontent.com/16021269/122761783-e5f98c00-d29c-11eb-9ee4-c5c5e479327f.png)

Reply:
Maybe it could be named "Thumbnail" or "Preview"?

Reply:
IMHO it looks best without any name